public/js/index.js

```diff
@@ -1679,12 +1679,22 @@ $(function () {
 function showDeletePopup() {
     var $this = $(this);
     var filter = "";
+
+    // Replace content to repositoy name
+    var content = $('.content').text();
+    content     = content.replace("%s",$this.data('repo-name'));
+    $('.content').html(content);
+
     if ($this.attr("id")) {
         filter += "#" + $this.attr("id")
     }
 
     $('.delete.modal' + filter).modal({
         closable: false,
+        onDeny: function() {
+            content     = content.replace($this.data('repo-name'), "%s");
```

Comment:
What if repo name will match any word in that message?

Reply:
if you look into the code, first time symbol `%s` will be replace  with repository name `$this.data('repo-name')` and then if cancel to remove repository the repository name will be replace with `%s`

Reply:
But what is if repository name will be `operation`? ;)

Reply:
Just save message contents before replacing in variable and restore it back later on

Reply:
@lafriks sorry i'm not understand with your question ( But what is if repository name will be operation?) 

i think not necessary to save message content in variabel, because is same. the content message after deleted repository is will be reset to default and tested after deleted repository or cancel delete repository, the message content is works